src/python/pants/option/global_options.py

```diff
@@ -183,6 +183,11 @@ def register_bootstrap_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--owner-of', type=list, default=[], metavar='<path>',
+             help='Select the targets that own these files.'
+                  'This is the third target calculation strategy along with the --changed'
+                  'options and specifying the targets directly. These three types of target'
+                  'selection are mutually exclusive.')
```

Comment:
nit: add spaces to the end of the lines so that when they're concatenated, they'll have separations.